src/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ include(CheckVariableExists)
 include(CheckTypeSize)
 include(CheckLibraryExists)
 include(CMakeDetermineCCompiler)
-include(FindPkgConfig)
```

Comment:
Should this also be removed? (not sure)
`CMakeFiles/Makefile.cmake:  "/usr/share/cmake/Modules/FindPkgConfig.cmake"`